lib/index.js

```diff
@@ -343,13 +356,8 @@ exports.renderFile = function(path, options, fn){
 
   options = options || {};
 
-  var key = path + ':string';
-
   options.filename = path;
-  var str = options.cache
-    ? exports.cache[key] || (exports.cache[key] = fs.readFileSync(path, 'utf8'))
-    : fs.readFileSync(path, 'utf8');
-  return exports.render(str, options);
+  return exports.render(null, options);
```

Comment:
Maybe it would be better if this did:

```js
return handleTemplateCache(options)(options);
```

It's not much duplicated code, and it keeps `exports.render` from needing to accept `null` for the string.  We could then add a check in `exports.render` that throws if `str === null` to catch that odd corner case.  I'd hate to end up having to support it forever.